schemas/creativeWork.schema.tpl.json

```diff
@@ -0,0 +1,123 @@
+        "legalPerson"
+      ]
+    },
+    "author": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several authors (person or organization) that contributed to the production and publication of this creative work.",
+      "_linkedCategories": [
+        "legalPerson"
+      ]
+    },
+    "citedPublication": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add citations or references from this work to another creative work",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DOI",
+        "https://openminds.ebrains.eu/core/ISBN"
+      ]
+    },
+    "copyright": {
+      "_instruction": "Add the copyright information of this creative work.",
+      "_embeddedTypes": [
+        "https://openminds.ebrains.eu/core/Copyright"
+      ]
+    },
+    "dateCreated": {
+      "type": "string",
+      "_formats": [
+        "date"
+      ],
+      "_instruction": "Enter the date on which the creative work was created."
+    },
+    "dateModified": {
+      "type": "string",
+      "_formats": [
+        "date"
+      ],
+      "_instruction": "Enter the date on which the creative work was last modified."
+    },
+    "datePublished": {
+      "type": "string",
+      "_formats": [
+        "date"
+      ],
+      "_instruction": "Enter the date when this version of the creative work was published or uploaded to a preprint server."
+    },
+    "digitalIdentifier": {
+      "_instruction": "Add the globally unique and persistent digital identifier of this creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DOI"
+      ]
+    },
+    "editor": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or more people who edited the creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/Person"
+      ]
+    },
+    "funding": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all funding information of this creative work version.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/Funding"
+      ]
+    },
+    "name": {
+      "type": "string",
+      "_instruction": "Enter the full title of this creative work."
+    },
+    "keyword": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add controlled or suggest new terms as keywords to this creative work",
+      "_linkedCategories": [
+        "studyTarget"
+      ]
+    },
+    "license": {
+      "_instruction": "Add the license for this creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/License"
+      ]
+    },
+    "publisher": {
+      "_instruction": "Enter the person or organisation who/which published this version of the creative work.",
+      "_linkedCategories": [
+        "legalPerson"
```

Comment:
should a journal be also an option here or is this registered as organization then (if yes why do we need periodical schema in addition) ?

Reply:
a journal is not a publisher, e.g. the journal Nature is published by the organization Springer-Nature

Reply:
okay. I was wondering about that. Thanks for clarifying